static/bidder-params/flipp.json

```diff
@@ -0,0 +1,41 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "title": "Flipp Adapter Params",
+  "description": "A schema which validates params accepted by the Flipp adapter",
+  "type": "object",
+  "properties": {
+    "publisherNameIdentifier": {
+      "type": "string",
+      "minLength": 1,
+      "description": "Publisher Name Identifier"
+    },
+    "creativeType": {
+      "type": "string",
+      "enum": ["NativeX", "DTX"]
+    },
+    "siteId": {
+      "type": "integer"
+    },
+    "zoneIds": {
+      "type": "array",
+      "items": {
+        "type": "integer"
+      }
+    },
+    "userKey": {
+      "type": "string",
+      "format": "uuid"
+    },
+    "ip": {
+      "type": "string"
+    },
+    "options": {
+      "type": "object"
```

Comment:
can you please add detailed schema over here? Based on your test jsons here is what it may look like please add any other fields that you are expecting as part of this object:-

```
"options": {
      "type": "object",
      "properties": {
        "startCompact": {
          "type": "boolean"
        }
      }
    }
```